### PR TITLE
CM Breaking changes in 6.7

### PR DESCRIPTION
**Pull request by ph**
Add a note concerning the breaking change of 6.7 for Central Management
and mention the migration tool.

### CODE REVIEW

File: libbeat/docs/breaking.asciidoc [hunk 1]
```diff
@@ -29,6 +30,14 @@ upgrade the Beats to version 7.0. {see-relnotes}
 
 include::./field-name-changes.asciidoc[]
 
+[[breaking-changes-6.7]]
+=== Breaking changes in 6.7
+
+The Central management document format used by Kibana has changed. As a result of the template
+changes beats and tags created prior to 6.7 will not work with the new UI until their are moved to
+the new format, we have created a one time [tool]() to help you migrate your data from
```
Comment (ph) [inline libbeat/docs/breaking.asciidoc]: @kvch I will use your link here.

File: libbeat/docs/breaking.asciidoc [hunk 2]
```diff
@@ -29,6 +30,14 @@ upgrade the Beats to version 7.0. {see-relnotes}
 
 include::./field-name-changes.asciidoc[]
 
+[[breaking-changes-6.7]]
+=== Breaking changes in 6.7
+
+The Central management data structure used by Kibana to store the information has changed. As a result of the
```
Comment (dedemorton) [inline libbeat/docs/breaking.asciidoc]:
```suggestion
The data structure used to store Beats central management data in Kibana has changed. As a result of the
```

File: libbeat/docs/breaking.asciidoc [hunk 3]
```diff
@@ -29,6 +30,14 @@ upgrade the Beats to version 7.0. {see-relnotes}
...
+template changes, enrolled beats and tags created prior to 6.7 will not work with the new
```
Comment (dedemorton) [inline libbeat/docs/breaking.asciidoc]:
```suggestion
template changes, enrolled Beats and configuration tags created prior to 6.7 will not work until you migrate them to the new format.
```

File: libbeat/docs/breaking.asciidoc [hunk 4]
```diff
@@ -29,6 +30,14 @@ upgrade the Beats to version 7.0. {see-relnotes}
 
 include::./field-name-changes.asciidoc[]
 
+[[breaking-changes-6.7]]
+=== Breaking changes in 6.7
+
+The data structure used to store Beats central management data in Kibana has changed. As a result of the
+template changes, enrolled Beats and configuration tags created prior to 6.7 will not work until you migrate them to the new format.
+UI until their are moved to the new format, we have created a one time [tool](https://github.com/elastic/migrate-management-beats/releases) to help you migrate your data from
```
Comment (dedemorton) [inline libbeat/docs/breaking.asciidoc]:
```suggestion
```suggestion
To migrate to the new format, use the https://github.com/elastic/migrate-management-beats/releases[migration tool] we've created.
```
```

Comment (dedemorton) [inline libbeat/docs/breaking.asciidoc]:
```suggestion
To migrate to the new format, use the https://github.com/elastic/migrate-management-beats/releases[migration tool] we've created.
```

File: libbeat/docs/breaking.asciidoc [hunk 5]
```diff
@@ -29,6 +30,14 @@ upgrade the Beats to version 7.0. {see-relnotes}
...
+6.6 to 6.7, refer to the [documentation](https://github.com/elastic/migrate-management-beats/blob/master/README.md) for instruction. {see-relnotes}
```
Comment (dedemorton) [inline libbeat/docs/breaking.asciidoc]:
```suggestion
For more information, see the https://github.com/elastic/migrate-management-beats/blob/master/README.md[documentation] about the tool. {see-relnotes}
```

**Pull request merged**